control-plane/agents/src/bin/ha/cluster/nodes.rs

```diff
@@ -52,11 +52,11 @@ impl NodeList {
 
         info!(node.id=%node, %path, "Sending switchover for path");
 
-        if let Some(uri) = uri {
+        if let Some(socket) = uri {
             info!(node.id=%node, %path, "Sending switchover for path");
-
-            mover.switchover(uri.to_string(), path.clone()).await?;
-            failed_path.insert(path, uri);
+            //let uri = uri.to_string().parse::<Uri>().unwrap();
```

Comment:
Is this comment needed?

Reply:
Removed it

Reply:
That was not a browser issue. I had removed some commits by mistake. I have pushed those commits again. All changes Tiago requested are there now.